### PR TITLE
use type WARN for warning

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

File: daemon/daemon.go [hunk 1]
```diff
@@ -1228,11 +1228,11 @@ func checkKernel() error {
 	// the circumstances of pre-3.8 crashes are clearer.
 	// For details see http://github.com/docker/docker/issues/407
 	if k, err := kernel.GetKernelVersion(); err != nil {
-		log.Infof("WARNING: %s", err)
+		log.Warnf("WARNING: %s", err)
```
Comment (LK4D4) [inline daemon/daemon.go]: Can we remove word `WARNING` everywhere? because it is sorta redundant now

Comment (crosbymichael) [inline daemon/daemon.go]: +1 to removing warning form the message content.

**Comment by hqhq** [comment]
@LK4D4 @crosbymichael 
Updated, please take a look.

**Comment by hqhq** [comment]
ping @LK4D4 @crosbymichael

**Comment by LK4D4** [comment]
There is some incosistencies between `Errorf` and `Warnf` usage, but I think it's okay for now.
LGTM

**Comment by icecrime** [comment]
LGTM

**Comment by icecrime** [comment]
I don't think any docs change is required here.

**Pull request merged**